flixel/tweens/FlxEase.hx

```diff
@@ -50,7 +54,7 @@ class FlxEase
 
 	public static inline function quadInOut(t:Float):Float
 	{
-		return t <= .5 ? t * t * 2 : 1 - (--t) * t * 2;
+		return t <= .5 ? quadIn(t) * 2 : 1 - (--t) * t * 2;
```

Comment:
is `1 - (--t) * t * 2` just a fancy way of saying `quadOut((t - 0.5) * 2)`? changing would be more readible and it's all inlined to reduced math expressions by the compiler anyway

Reply:
also I believe this should be `quadIn(t * 2)` not `quadIn(t) * 2`. so please visually verify each function that was changed

Reply:
Here's an example of how I personally, visually verified the changes of your previous PR
https://github.com/Geokureli/flixel-tests/blob/main/Source/states/BounceEaseTestState2788.hx

if the equations were not changed blue and dark blue should always be touching, regardless of the mouse position, same with red and green